paasta_tools/utils.py

```diff
@@ -3268,8 +3268,8 @@ def set_paasta_print_file(file: Any) -> Iterator[None]:
 
 
 def paasta_print(*args: Any, **kwargs: Any) -> None:
-    f = kwargs.pop("file", sys.stdout)
-    f = getattr(TLS, "paasta_print_file", f)
+    f = kwargs.pop("file", sys.stdout) or sys.stdout
+    f = getattr(TLS, "paasta_print_file", f) or sys.stdout
```

Comment:
I think we got this wrong -- should be `or f` instead of `or sys.stdout`

Reply:
To be clear (since github is showing more than just the line I highlighted) I mean the second of these -- the `getattr`

Reply:
ah right